index.html

```diff
@@ -1023,150 +1026,66 @@
-making sure that:
-
-* the [=identifiers=] used in the [=data=] are under the direct and exclusive control of
-  the [=first party=] who is prevented by strict controls from matching the identifiers with the
-  data;
-
-* when these [=identifiers=] are shared with a [=third party=], they are made unique
-  to that [=third party=] such that if they are shared with more than one
-  [=third party=] these cannot then match them up with one another;
-
-* there is a strong level of confidence that no [=third party=] can match the [=data=]
-  with any data other than that obtained through interactions with the [=first
-  party=];
+<h4>Pete's Funtime Reference Corral</h4>
+
+* <dfn data-lt="data">personal data</dfn>
+* <dfn data-lt="data processor">service provider</dfn>
+* <dfn data-export="">right to access</dfn>
+* <dfn data-export="" data-lt="right to erase">right to erase</dfn>
+* <dfn data-export="" data-lt="right to portability" data-local-lt="portability">right to port</dfn>
+* <dfn data-export="">right to correct</dfn>
+* <dfn data-export="">right to be free from automated decision-making</dfn>
+* <dfn data-export="" data-lt="right to object|right to withdraw consent|right to restrict use">right to object, withdraw consent, and restrict use</dfn>
+* <dfn>de-identified</dfn>
+* [=third party=]
+* [[?Privacy-Concerned]]
 
-* any [=third party=] receiving such [=data=] is barred (eg. based on legal terms)
-  from sharing it further;
-
-* technical measures exist to prevent re-identification or the joining of
-  different data sets involving this [=data=], notably against timing or
-  k-anonymity attacks; and
-
-* there exist contractual terms between the [=first party=] and [=third party=]
-  describing the limited [=purpose=] for which the data is being shared.
-
-Note that [=controlled de-identified data=], on its own, is not sufficient to render [=data processing=]
-[=appropriate=].
-
-### Rights {#data-rights}
+</aside>
 
-[=People=] retain certain rights over data about themselves, and these rights should be facilitated
-by their [=user agent=] and the [=parties=] they're interacting with. While data rights alone are
-not sufficient to satisfy all [=privacy=] principles for the Web, they do support self-determination
-and help improve accountability. Such rights include:
+<aside class="issue">
+  This section has not been edited for narrative.
+</aside>
 
-* The <dfn data-export="">right to access</dfn> [=data=] about oneself.
+<div class="practice">
 
-This right includes being able to discover data about oneself (implying no databases are kept
-secret) and to review
-what information has been collected or inferred.
+<p><span class="practicelab" id="user-agent-data-sharing">
+User agents should only share and present information about the user that
+the user can reasonably anticipate being shared, and which directly relate to the users overt, immediate goals.
```

Comment:
Some folks pointed out that "directly relate to ... overt, immediate goals" is probably too limiting. Users have privacy goals that are longer-term than immediate, for example avoiding the need to log in with PII, and we shouldn't have this document claim that short-term privacy goals always trump long-term ones.

Reply:
I do not think i follow.  The "users overt, immediate goals" text is about what the user intends to do on the page or how they intend to interact w/ the site, not the user's privacy goals. Am i misunderstanding the concern?